packages/streamid/src/stream-type.ts

```diff
@@ -3,7 +3,7 @@ const registry: Record<string, number | undefined> = {
   'caip10-link': 1,
 };
 
-export function indexByName(name: string): number {
+function indexByName(name: string): number {
```

Comment:
Can we call these `idByName` and `nameById`?

Reply:
`typeByName`, `nameByType`?

Reply:
It feels a little weird to me to call `StreamType.nameByType()` and pass it a number.  It seems like a function on an object called `StreamType` that says it takes a 'type' should take a `StreamType` instance.  But I don't feel strongly and still like using 'type' better than 'index'

Reply:
Ah, maybe `codeByName` and `nameByCode`.

Reply:
But I'm quite ambivalent at this point :)